Bottom: e9b7748412f26de493855d399ba84e4c2700ff65
Top:    263f1b2ef480c8aefe66dc0e77faccbca2d820bb
Author: scott snyder <sss@karma>
Date:   2020-02-03 10:06:05 -0500

.gitignore


---

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..b25c15b
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+*~
